client/src/components/Round/Round.jsx

```diff
@@ -72,6 +76,12 @@ const ROUND_UPDATED_SUBSCRIPTION = gql`
   ${ROUND_RESULT_FRAGMENT}
 `;
 
+// Events sorted by best don't need forecast view.
+// Fewest moves is currently unsupported
+export function forecastViewDisabled(round) {
```

Comment:
Nit: let's move this to `result.js` also to keep the logic together, and let's revert as `forecastViewEnabled` (so that we don't need double negation as in `!forecastViewDisabled(round)`).